classes/controller/AdminController.php

```diff
@@ -436,7 +436,9 @@ public function __construct($forceControllerName = '', $default_theme_name = 'de
         $this->bo_css = ((Validate::isLoadedObject($this->context->employee)
             && $this->context->employee->bo_css) ? $this->context->employee->bo_css : 'admin-theme.css');
 
-        if (!@filemtime(_PS_BO_ALL_THEMES_DIR_.$this->bo_theme.DIRECTORY_SEPARATOR.'css'.DIRECTORY_SEPARATOR.$this->bo_css)) {
+        $adminThemeCSSFile = _PS_BO_ALL_THEMES_DIR_.$this->bo_theme.DIRECTORY_SEPARATOR.'css'.DIRECTORY_SEPARATOR.$this->bo_css;
+
+        if (file_exists($adminThemeCSSFile) && !@filemtime($adminThemeCSSFile)) {
```

Comment:
You can remove filemtime() then. It has the same purpose here, but it is slightly faster.

Reply:
Done! Thanks for review :+1: